docs/src/views/Theming.tsx

```diff
@@ -208,6 +208,82 @@ export default () => (
       <NavLink to="components/provider">Provider</NavLink> at the root of your app.
     </p>
 
+    <Header as="h2" content="Animations" />
+    <p>
+      Another important part of the theming in Stardust are the <code>animations</code>. You can
+      define the animations in a very similar way as you would define them using css, by providing a
+      keyframe, as well as the other animation properties: duration, delay, timingFunction etc.
+    </p>
+
+    <p>
+      You can define the <code>animations</code> in a part of your render tree using the{' '}
+      <NavLink to="components/provider">Provider</NavLink>.
+    </p>
+    <p>
+      This is done with the Provider's <code>theme</code> prop. The animations are then are applied
+      based on their name by using the <NavLink to="components/transition">Transition</NavLink>{' '}
+      component, or the <code>animation</code> property available on all UI component. Here's how we
+      can define the <code>animations</code> and use them in our components.
+    </p>
+    <ExampleSnippet
+      value={[
+        `<Provider`,
+        `  theme={{`,
+        `    animations: {`,
+        `      spinner: {`,
+        `        keyframe: {`,
+        `          from: {  transform: 'rotate(0deg)' },`,
+        `          to: { transform: 'rotate(360deg)' },`,
+        `        },`,
+        `        duration: '5s',`,
+        `        iterationCount: 'infinite'`,
+        `     },`,
+        `    },`,
+        `  }}`,
+        `>`,
+        `  <div>`,
+        `    <Transition animationName="spinner"><Icon name="user" circular /></Transition>`,
+        `    <Icon name="book" animation="spinner" circular/>`,
+        `  </div>`,
+        `</Provider>`,
+      ].join('\n')}
+      render={() => (
+        <div>
+          <Transition animationName="spinner">
```

Comment:
Thanks for the suggestion, I will rename the Transition component to Animation, as that is the work we use for describing this behavior everywhere else. Transition was just picked up because of SUIR. Then we can safely rename the animationName prop to name. I will add this changes to this PR too.